api2/pkg/apps/cms/case_store.go

```diff
@@ -71,15 +82,13 @@ func (s *Store) List(ctx context.Context, listOptions ListOptions) (*CaseList, e
 	return &ret, nil
 }
 
-func (s *Store) Update(ctx context.Context, kase *Case) error {
+func (s *CaseStore) Update(ctx context.Context, kase *Case) error {
 	_, err := s.collection.UpdateOne(ctx, bson.M{
 		"id": kase.ID,
 	}, bson.M{
 		"$set": bson.M{
 			"description": kase.Description,
 			"done":        kase.Done,
-			"parentId":    kase.ParentID,
-			"teamId":      kase.TeamID,
```

Comment:
Why were these two lines removed?

Reply:
I figured we wanted to make these readonly (not able to be updated)